mozilla_django_oidc_db/backends.py

```diff
@@ -93,6 +69,18 @@ def __init__(self, *args, **kwargs) -> None:
         # AbstractUser.
         self.UserModel = cast(AbstractUser, get_user_model())
 
+    @property
+    def config(self):
```

Comment:
a type hint for the return value would help in downstream calls to figure out what is actually available